static/app/components/onboardingWizard/task.tsx

```diff
@@ -71,7 +71,9 @@ function Task({router, task, onSkip, onMarkComplete, forwardedRef, organization}
     }
 
     if (task.actionType === 'app') {
-      navigateTo(`${task.location}?onboardingTask`, router);
```

Comment:
Is it OK that the `onboardingTask` query key has gone away?

Reply:
@evanpurkhiser tbh I don't even understand what it does, it's not like we said `referrer=onboarding_task`, this seemed like a bug more than anything lol

Reply:
I think `onboardingTask` may have at one point cased it to open the onboarding task list?

Reply:
@evanpurkhiser Not sure why you'd do that for an item coming from the onboarding task list. Anyways, no code is referencing it anymore.

Reply:
sgtm

Reply:
oh no idea lol